### DISCUSSION TITLE
Accidentally set all users as Staff

### DISCUSSION THREAD

**Discussion by nbanderso** [babybuddy/babybuddy#603] [Support]
Hello,

As I was exploring Baby Buddy, I accidentally set all users as "Staff."  Now every user has limited permissions and there doesnt appear to be a way to revert this back.  

How can I deselect myself as a user when I no longer have the permissions?  

For reference, I am using the Home Assistant Add-on to host Baby Buddy.  

Thank you!

**Comment by cdubz** [comment]
You can create users from the command line. See https://docs.baby-buddy.net/user-guide/managing-users/#creating-a-user-from-the-command-line

**Reply by nbanderso** [reply]
Thanks @cdubz 

Any idea how I access the command line when I am hosting Baby Buddy on my Home Assistant?
